blog/2022-08-21-welcome/index.md

```diff
@@ -2,9 +2,9 @@
 slug: welcome
 title: Welcome
 authors: [afflom, jpower432, sabre1041, usrbinkat]
-tags: [uor, hello]
+tags: [uor, hello, emporous]
```

Comment:
Remove reference to `uor`

Reply:
Wasn't sure if we wanted to keep it for historical reasons. I have removed it.